server/src/pages/shot/time-diff.js

```diff
@@ -24,6 +24,12 @@ exports.TimeDiff = class TimeDiff extends React.Component {
     }
   }
 
```

Comment:
I've replied on IRC, but putting it here too. There's some standardization happening, but I think it won't be suitable for a website any time soon (zibi would know for sure): https://bugzilla.mozilla.org/show_bug.cgi?id=1270140

Using external libraries is not ideal, there are concerns about quality and control about the localization living there.

Concatenating is very frequently a path to disaster, especially when it comes to plural, gender, and declensions.
Do we have an idea of how many variations we have? e.g. "expired X seconds|minutes|hours|days|weeks|months|years ago", "expires in X seconds|minutes|hours|days|weeks|months|years"? 

I know it's a step backward in terms of usability, but is there any chance to have "expire(s|d) on X", with X being a localized date?